ONSdigital/rm-collection-exercise-service
# Revert "Added an integration test to test receiving sample upload finished messages (#63)"

# Motivation and Context
The original commit pioneered use of a simple Java client to send messages from services instead of Spring integration.  However there were issues around configuration of this client in cloud foundry. The commit is being backed out with a view to resolving the issue at a later date.

# What has changed
This reverts commit 3f442f0205297cd4040553ad4c8860f82f906ba4.

# How to test?
- Deploy updated service to concourse
- Ensure that all is green (especially ci-rasrm-acceptance-tests)
(problem cannot be reproduced locally - only occurs in Cloud Foundry)

# Links
[Concourse](https://concourse.onsdigital.uk/teams/rmras)
[Concourse ci-rasrm-acceptance-tests](https://concourse.onsdigital.uk/teams/rmras/pipelines/rasrm/jobs/ci-rasrm-acceptance-tests)
